integration-tests/tests/all_test.go

```diff
@@ -26,18 +26,21 @@ const (
 )
 
 func TestCameraServer(t *testing.T) {
+	ctxTimeoutTests, cancel := context.WithTimeout(context.Background(), 30*time.Second)
```

Comment:
I always have weird peripheral changes on these PRs lol sorry

This is just because I found that integration tests hangs when encountering errors sometimes, so we just want to set a maximum of 30 seconds for them.